checker/tests/resourceleak/FilesTest.java

```diff
@@ -0,0 +1,20 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+import org.checkerframework.checker.mustcall.qual.MustCall;
+
+public class FilesTest {
+
+  void bad(Path p) throws IOException {
+    // error: [required.method.not.called]
+    Stream<Path> s = Files.list(p);
+  }
+
+  void good(Path p) throws IOException {
+    // TODO: Programmers should not be required to write this `@MustCall("close")`.
```

Comment:
Can we file a separate issue on this?  This will be a bad developer experience

Reply:
@msridhar I agree.  I won't merge this pull request until the issue is resolved.  I have already communicated to @smillst about it.

Reply:
I've fixed this here: https://github.com/typetools/checker-framework/pull/5334